streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+                output_doc['_rev'] = doc['_rev']
+                if 'wiki_page' in doc['payload']:
+                    output_doc['wiki_page'] = doc['payload']['wiki_page']
+                if 'min_weight' in doc['payload']:
+                    output_doc['min_weight'] = doc['payload']['min_weight']
+                if 'max_weight' in doc['payload']:
+                    output_doc['max_weight'] = doc['payload']['max_weight']
+                if 'class' in doc['payload']:
+                    output_doc['class'] = doc['payload']['class']
+                if 'diet' in doc['payload']:
+                    output_doc['diet'] = doc['payload']['diet']
+                # Submit a tuple in each iteration:
+                submit(output_doc)
+    ```
+1. Select the `Edit Output Schema` below the code editor.
+1. Add the following attribute names and their types using the **Add Attribute** button.
+   Press **Save** then **Close** when finished.
+
+    Name | Type
+    --- | --- 
+    _id | Text
+    _rev | Text
+    wiki_page | Text
+    min_weight | Number
+    max_weight | Number
+    class | Text
+    diet | Text
+
+    ![Output Schema for Code Operator](attributes-code-streams-designer.png)
+
+1. Select the floppy disk icon in the Stream Designer toolbar to save the flow.
+1. Drag and drop the Db2 Warehouse on Cloud operator under **Targets** on to the canvas.
+1. Select the Db2 Warehouse on Cloud operator and a right pane will open for adding connection details.
+![Db2 operator in Streams Designer](db2-operator-streams-designer.png)
+1. Click `Add Connection` and select the Db2 Warehouse on Cloud instance that was previously created.
+Select **Create** to finish.<br/>
+**Note:** If `Your service instances in IBM Cloud` is empty, you can [grab the JDBC connection string](../db2/connection_details.md)
+and create a connection manually. 
+![Add Db2 connection](db2-operator-add-connection.png)
+1. Click on the icon under **Schema/Table** to select the Db2 table.
+1. Select your schema and `ANIMALS` table, then press **Select**.
+![Select schema in Db2 operator](select-schema-table-db2-operator.png)
+1. Link the two operators together by dragging your mouse cursor from the output port of the `Code` operator to the input 
+port of the Db2 operator.
+![Link operators in Streams Designer](link-operators-streams-designer.png)
+1. Select **Map Schema** in the Db2 operator's right pane.
+![Select Map Schema in Db2 operator](map-schema-db2-operator.png)
+1. Assign the attributes previously created in the `Code` operator to it's equivalent Db2 target column, and then press **Save**.
+![Map attributes to Db2 target columns](assign-attributes-db2-operator.png)
+1. Press the play icon in the Stream Designer toolbar to save and run the streams flow.
+![Play icon in Streams Designer](play-icon-stream-designer.png)
+1. The Status indicator in the Metrics page will change from "Stopped" to "Starting", and then to “Running”.
+![](starting)
+![](running)
```

Comment:
I don't think there are any images for starting/running so these lines can be removed.